src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,142 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService reg;
+
+    @BeforeEach
+    void setUp() {
+        reg = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void userIsOlder_Ok() {
+        User expected = new User(29892234L, "test_user22", "test_password1", 20);
```

Comment:
do not set id to user
```suggestion
        User expected = new User("test_user22", "test_password1", 20);
```